sentry_sdk/utils.py

```diff
@@ -611,7 +610,7 @@ def serialize_frame(
         )
 
     if include_local_variables:
-        rv["vars"] = copy(frame.f_locals)
+        rv["vars"] = frame.f_locals.copy()
```

Comment:
why this change?

Reply:
[This commit](https://github.com/getsentry/sentry-python/pull/3200/commits/49b4cf328e115d5639e8bdc7b0978a8abf7c2fb6) has the details.

Reply:
https://github.com/getsentry/sentry-python/commit/49b4cf328e115d5639e8bdc7b0978a8abf7c2fb6

Reply:
^ Basically, `frame.f_locals` is no longer a `dict` in 3.13